NOTICE

```diff
@@ -3,6 +3,13 @@ Copyright 2021 Splunk Inc.
 
 Third-party Software Attributions:
 
+Library: beautifulsoup4
+Version: 4.9.1
+License: MIT
+Copyright 2004-2017 Leonard Richardson
+Copyright 2004-2019 Leonard Richardson
```

Comment:
isn't the first of these redundant? not sure if it was just copied from somewhere in which case it's fine to keep as is since this is a legal thing

Reply:
Oher apps which contains the bs4 has this type of details in the NOTICE file. I have copied from [here](https://github.com/splunk-soar-connectors/agari/blob/next/NOTICE)